products/sle/main.pm

```diff
@@ -699,6 +699,10 @@ elsif (get_var("NFV")) {
     boot_hdd_image();
     load_nfv_tests();
 }
+elsif (get_var('TRINITY')) {
+    boot_hdd_image();
+    loadtest "kernel/trinity";
```

Comment:
Maybe you should put this in main_ltp? even though it is not LTP, it is related to the other kernel tests. Then add the change_kernel/kotd module the same as the LTP tests.

Reply:
+1